qa/tasks/vstart_runner.py

```diff
@@ -1338,6 +1343,10 @@ def exec_test():
     if opt_create_cluster_only:
         return
 
+    global FSID
```

Comment:
`FSID` is used at Line 950, maybe moving `FSID` there would be better?

Reply:
It's more suited here. We collect FSID as early as it can be.

Reply:
Okay